docs/DeveloperGuide.md

```diff
@@ -1,29 +1,136 @@
 # Developer Guide
 
+* [Acknowledgements](#acknowledgements)
+* [Setting up, getting started](#setting-up-getting-started)
+* [Design](#design)
+    * [Architecture](#architecture)
+    * [UI Component](#UI-Component)
```

Comment:
The following links do not work:
UI Component
SaveGoal Component
Withdrawal Limit Component
Transaction Component
Account related Feature
saveGoal related Feature
Transaction Related Feature

Perhaps you should fix them?